### DISCUSSION TITLE
[Kanban] Clicking internal links in cards should open the tiddler normally

### DISCUSSION THREAD

**Discussion by AshRubigo** [BurningTreeC/tiddlywiki-muuri#13] [Ideas]
Edit: Fixed by @BurningTreeC for version 0.1.13.

Currently clicking an internal link in cards opens the tiddler *within* the list the card is in, and even more odd is that it is also *added* to that list.

Instead the expected behaviour is that the tiddler would open just like any other per the current in the story view.

**Comment by BurningTreeC** [comment]
@AshRubigo  thanks for your suggestion
I will add a configuration option that allows changing that behavior.

Thank you!
